charts/fleet-agent/100.0.4+up0.3.10-rc5/Chart.yaml.orig

```diff
@@ -6,8 +6,8 @@ annotations:
   catalog.cattle.io/permits-os: linux,windows
   catalog.cattle.io/release-name: fleet-agent
 apiVersion: v2
-appVersion: 0.3.10-rc4
```

Comment:
this file shouldn't exist? Seems like there was an error when applying the patch file in `packages/fleet/fleet-agent/generated-changes/patch`